File: PPA/DailyFlash/August/19August/Program5.c

#include<stdio.h>

void main() {

	int ang1, ang2, ang3;

	printf("Enter the three angle of a triangle: ");
	scanf("%d%d%d",&ang1,&ang2,&ang3);

	if(ang1+ang2+ang3==180)
		printf("The triangle with angle %d %d & %d is a valid one.\n",ang1,ang2,ang3);
	else
		printf("The triangle with angle %d %d & %d is not a valid one.\n",ang1,ang2,ang3);
}

File: Java/DailyFlash/August/19Aug/Program5.java

import java.util.Scanner;

class Triangle {

	public static void main(String[] args) {

	Scanner sc = new Scanner(System.in);
	int ang1, ang2, ang3;

	System.out.print("Enter the three angle of a triangle: ");

	ang1= sc.nextInt();
	ang2= sc.nextInt();
	ang3= sc.nextInt();

	if(ang1+ang2+ang3==180)
		System.out.println("The triangle with angle "+ang1+", "+ang2+" & "+ang3+" is a valid one");
	else
		System.out.println("The triangle with angle "+ang1+", "+ang2+" & "+ang3+" is not a valid one");
	}
}

